dirs/dirs.go

```diff
@@ -187,6 +191,10 @@ func SetRootDir(rootdir string) {
 
 	SnapStateFile = filepath.Join(rootdir, snappyDir, "state.json")
 
+	SnapCacheDir = filepath.Join(rootdir, snappyDir, "cache")
```

Comment:
Why not `/var/cache/snapd`?

Reply:
I'd like it to be `/var/cache/snapd` as well, but that would mean adding that to `writable-paths`, and I don't know the impact on packaging elsewhere (is `/var/cache/{package}` universal?); keeping it under `snappyDir` makes it a no-brainer.